operator/config/default/kustomization.yaml

```diff
@@ -46,20 +45,20 @@ patchesStrategicMerge:
 # the following config is for teaching kustomize how to do var substitution
 vars:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER' prefix.
-- name: CERTIFICATE_NAMESPACE # namespace of the certificate CR
-  objref:
-    kind: Certificate
-    group: cert-manager.io
-    version: v1
-    name: serving-cert # this name should match the one in certificate.yaml
-  fieldref:
-    fieldpath: metadata.namespace
-- name: CERTIFICATE_NAME
-  objref:
-    kind: Certificate
-    group: cert-manager.io
-    version: v1
-    name: serving-cert # this name should match the one in certificate.yaml
+#- name: CERTIFICATE_NAMESPACE # namespace of the certificate CR
```

Comment:
please remove if not needed

Reply:
we leave all unused file commented in the generated part of kubebuilder, shall I remove all?

Reply:
if we do not need it I would say yes